tiledb/sm/consolidator/fragment_consolidator.cc

```diff
@@ -624,7 +620,7 @@ FragmentConsolidator::create_buffers(
   }
 
   // Success
-  return {buffers, buffer_sizes};
+  return std::move(cw);
```

Comment:
The central performance improvement is to move the return value. This could have happened with vectors, but the new class allows verification by making a small modification to turn on the copy constructor and use it in this return statement.